condacolab.py

```diff
@@ -125,18 +152,16 @@ def install_from_url(
     if sitepackages not in sys.path:
         sys.path.insert(0, sitepackages)
 
-    print("🩹 Patching environment...")
     env = env or {}
     bin_path = f"{prefix}/bin"
-    if bin_path not in os.environ.get("PATH", "").split(":"):
-        env["PATH"] = f"{bin_path}:{os.environ.get('PATH', '')}"
-    env["LD_LIBRARY_PATH"] = f"{prefix}/lib:{os.environ.get('LD_LIBRARY_PATH', '')}"
 
     os.rename(sys.executable, f"{sys.executable}.real")
     with open(sys.executable, "w") as f:
         f.write("#!/bin/bash\n")
-        envstr = " ".join(f"{k}={v}" for k, v in env.items())
-        f.write(f"exec env {envstr} {sys.executable}.real -x $@\n")
+        f.write(f"source {prefix}/etc/profile.d/conda.sh\n")
+        f.write("conda activate\n")
+        f.write(f"unset PYTHONPATH\n")
+        f.write(f"exec {bin_path}/python $@\n")
```

Comment:
Let's rewrite this in a cleaner way for readability.

We can use triple-quote strings and `textwrap.dedent()` to write it like this:


```suggestion
        f.write(
            dedent(
                f"""
                source {prefix}/etc/profile.d/conda.sh
                conda activate
                unset PYTHONPATH
                exec {bin_path}/python $@
                """
            ).lstrip()
        )
```